<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetTitle" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="statusTip">
       <string notr="true"/>
      </property>
      <property name="whatsThis">
       <string notr="true"/>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widgetTitle{background:#363636;}
QLabel#labelTitleText{color:rgb(255,255,255);padding:9px 0px 5px;}
QPushButton#pushButtonMin,QPushButton#pushButtonMax,QPushButton#pushButtonClose{
        border-radius:0px;
        color: #F0F0F0;
        background-color:rgba(0,0,0,0);
        border-style:none;
    }
QPushButton#pushButtonMin:hover,QPushButton#pushButtonMax:hover{
        background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(25, 134, 199, 0), stop:1 #5CACEE);
    }
QPushButton#pushButtonClose:hover{
        background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(238, 0, 0, 128), stop:1 rgba(238, 44, 44, 255));
    }</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="labelTitleImg">
         <property name="minimumSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="rc.qrc">:/image/title.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTitleText">
         <property name="text">
          <string>DouYuTv弹幕</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMin">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>最小化</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/image/min.png</normaloff>:/image/min.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMax">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>靠右侧</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/image/max1.png</normaloff>:/image/max1.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>退出</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/image/exit.png</normaloff>:/image/exit.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#widget{background-color:rgb(175, 197, 255)}
QPushButton#pushButton{
   /* 前景色 */
    color:rgb(0, 0, 0);
    /* 边框风格 */
    border-style:outset;
}
QLineEdit#lineEdit{
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background: rgb(134, 219, 255);}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>弹幕房间号：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>进入房间</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QToolBox" name="toolBox">
            <property name="styleSheet">
             <string notr="true">QToolBox#toolBox::tab {
background:rgb(175, 197, 255);
}
QScrollBar:vertical {
    width: 4px;
    background:rgba(175, 197, 255,0%);
    margin:0px,0px,0px,0px;
    padding-top:5px;
    padding-bottom:5px;
}

QScrollBar::handle:vertical,
{
    background:rgb(175, 197, 255);
}

QScrollBar::handle:vertical:hover,
{
    background: rgb(175, 197, 255);
}
QScrollBar::handle:vertical:hover
{
     background:rgb(175, 197, 255);
}

QScrollBar::sub-line:vertical, QScrollBar::add-line:vertical{
    width: 1px;
    height: 1px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="pageLive">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>328</width>
               <height>431</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QWidget#pageLive{
background-color:rgb(175, 197, 255);
}
</string>
             </property>
             <attribute name="label">
              <string>直播</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButtonGetRoom">
                <property name="styleSheet">
                 <string notr="true">QPushButton#pushButtonGetRoom{
   /* 前景色 */
    color:rgb(0, 0, 0);
    /* 边框风格 */
    border-style:outset;
}</string>
                </property>
                <property name="text">
                 <string>获取房间</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
